Source/SwiftLintFramework/Rules/Performance/FirstWhereRule.swift

```diff
@@ -16,7 +16,9 @@ public struct FirstWhereRule: CallPairRule, OptInRule, ConfigurationProviderRule
             "myList.first(where: { $0 % 2 == 0 })\n",
             "match(pattern: pattern).filter { $0.first == .identifier }\n",
             "(myList.filter { $0 == 1 }.suffix(2)).first\n",
-            "collection.filter(\"stringCol = '3'\").first"
+            "collection.filter(\"stringCol = '3'\").first",
+            "realm?.objects(User.self).filter(NSPredicate(format: \"email ==[c] %@\", email)).first",
+            "} else if let pause = timeTracker.pauses.filter(\"beginDate < %@\", beginDate).first {"
```

Comment:
can you update this to be a valid Swift snippet? (not starting with `} else` and with a full body `{}`)

Reply:
Oh, I didn't see this last time, just fixed it.